### ISSUE TITLE
Whether the network is generalizable

### ISSUE THREAD

**Issue by ss0912** [ryanhe312/STSSNet-AAAI2024#1]
I noticed that the checkpoints you provide contain the pkl file for each scenario, so do I need to retrain once for each scenario?

**Comment by ryanhe312** [comment]
Not necessarily. Our experiments are two-fold: we evaluate for single-scene (train and test on LE, ST, and SW separately) and cross-scene (train on LE, ST, and SW, test on AR) settings. The experiments on AR show the generalization capability of our model.

**Comment by ss0912** [comment]
Thank you very much for your answer. I would like to ask if your network can be used in UE?

**Comment by ss0912** [comment]
Thank you very much for your answer. I would like to ask if your network can be used in UE? Just like DLSS.

**Comment by ss0912** [comment]
I would also like to ask, how do you handle your network dataset?

**Comment by ryanhe312** [comment]
> Thank you very much for your answer. I would like to ask if your network can be used in UE? Just like DLSS.

Currently not available. We are still working on it.

> I would also like to ask, how do you handle your network dataset?

Do you mean dataset storage? We store the data as npy files.

**Comment by ss0912** [comment]
I mean how to process the images collected from unreal engine into npy files. I noticed that you only provide npy files for a few pre-processed scenes. How should I process my image data when adding scenes that I want to train myself to collect?

**Comment by ss0912** [comment]
I mean how to process the images collected from unreal engine into npy files. I noticed that you only provide npy files for a few pre-processed scenes. If I want to train my own collected scenes, how should I process my image data?

**Comment by ryanhe312** [comment]
You may refer to [ExtraNet](https://github.com/fuxihao66/ExtraNet) for the detailed data processing pipeline. Npy files are just for compressing.

**Comment by ss0912** [comment]
I have a question regarding "STSSNet". As I understand it, the input to STSSNet includes information from frames at t-5, t-3, and t-1, and the output is the prediction for frame t. Is my understanding correct?

**Comment by ryanhe312** [comment]
For extrapolation frames, yes. For super-resolution frames, inputs are t-4, t-2, and t. Please refer to the paper for details.

**Comment by ss0912** [comment]
I noticed that you have the training and test datasets in a compressed format. I know that in "compressedHR.XXXX.npy" is saved the content of the image in high resolution. I would like to know what exactly is saved in the "compressed.XXXX.NoWarp.npy" and "compressed.XXXX.Warp.npy" files in the NPY folder you provided? If you can answer this for me, I would be very grateful.
